### PR TITLE
Introduce script groups

### PR DESCRIPTION
**Pull request by nirs**
Speed up environment startup by running scripts in parallel.

With this change starting environment from scratch is 1.5 times faster.

### CODE REVIEW

File: test/README.md [hunk 1]
```diff
@@ -287,14 +287,20 @@ $ drenv delete example.yaml
...
+    - `name`: Optional worker name
+      `scripts`: Scripts to run by this worker
         - `file`: Script filename
-        - `args`: Optional argument to the script. If not specified the
+          `args`: Optional argument to the script. If not specified the
           script is run without any arguments.
-- `scripts`: Optional list of scripts to run after the profile scripts.
-    - `file`: Script filename
-    - `args`: Optional argument to the script. If not specified the
-      script is run without any arguments.
```
Comment (nirs) [inline test/README.md:297]: Need to fix, this is not yaml but markdown.
https://github.com/RamenDR/ramen/blob/a086c470112365daa0187fa6983e76eb2be50780/test/README.md#the-environment-file-format

Comment (nirs) [reply test/README.md:297]: Fixed in current version:
https://github.com/RamenDR/ramen/blob/a086c470112365daa0187fa6983e76eb2be50780/test/README.md#the-environment-file-format

File: test/ocm.yaml [hunk 2]
```diff
@@ -6,20 +6,23 @@
 profiles:
   - name: "hub"
     network: default
-    scripts:
-      - file: ocm-hub/start
-      - file: ocm-controller/start
+    workers:
+      scripts:
```
Comment (nirs) [inline test/ocm.yaml:10]: `- scripts:`

Comment (nirs) [reply test/ocm.yaml]: fixed in current version.

**Review by BenamarMk — APPROVED** [review]
LGTM

**Review by Madhu-1 — APPROVED** [review]

**Pull request merged**